CMakeLists.txt

```diff
@@ -447,7 +447,9 @@ include_directories("${CMAKE_CURRENT_BINARY_DIR}/include")
 # FIXME: External library includes
 include_directories(# For extlib-abc and extlib-constbv
                     ${PROJECT_SOURCE_DIR}/lib
+                    ${PROJECT_SOURCE_DIR}/lib/abc/src
                    )
+add_definitions(-DLIN64 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8 -DSIZEOF_INT=4)
```

Comment:
Can we use [`CHECK_TYPE_SIZE()`](https://cmake.org/cmake/help/latest/module/CheckTypeSize.html) to avoid hardcoding these?

Reply:
I implemented your idea.